webapp/package.json

```diff
@@ -76,7 +76,7 @@
         "eslint-plugin-react": "7.20.6",
         "eslint-plugin-react-hooks": "4.1.2",
         "file-loader": "6.1.0",
-        "jest": "26.4.2",
+        "jest": "27.0.2",
```

Comment:
Why have we kept it in both "devDependencies" and "dependencies"?

Reply:
This was done by a security bot.  I don't think it knows to remove redundant dependencies.